### PR TITLE
Icons

### PR DESCRIPTION
**Pull request by marcusfreeagent**

### CODE REVIEW

**Comment by marcusfreeagent** [comment]
[Icons 06: Documentation](https://trello.com/c/bSO8I37k/1712-icons-06-documentation)

File: docs/visuals/icons.md [hunk 1]
```diff
@@ -2,22 +2,169 @@
 categories: visuals
 layout: page
 title: Icons
+title: Iconography
+order: 4
 ---
 
-Icons
+<style type="text/css">
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: we probably want to look at how we handle page CSS outside of inline in the markdown file. there's a 'docs' CSS folder for general document stuff, like positioning of images etc. There _probably_ shouldn't be any styling that is only relevant to that one page i don't think so it'd be good to pull it out and make it available to other pages.

Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: lets look at CSS options for outside of the page

**Comment by craigcwaveformlabs** [comment]
we probably want to look at how we handle page CSS outside of inline in the markdown file. there's a 'docs' CSS folder for general document stuff, like positioning of images etc. There _probably_ shouldn't be any styling that is only relevant to that one page i don't think so it'd be good to pull it out and make it available to other pages.

File: docs/visuals/icons.md [hunk 2]
```diff
@@ -2,22 +2,169 @@
...
+  </p>
+</div>
+
+### What are icons?
+Icons are pictograms used to communicate a metaphor, often relating to an action or used with a label to support the meaning of something.
+
+We use them to help our users understand status and intent, although they can also be used for decoration.
+
+### Metaphors
+There are scenarios when an icon will appear in isolation and it's therefor important that its metaphor is clear and easy to interpret.
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: spelling? therefor

File: docs/visuals/icons.md [hunk 3]
```diff
@@ -2,22 +2,169 @@
...
+Inevitably people will interpreted icons in different ways, using team consensus is important to agree on the most appropriate design.
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: interpret

File: docs/visuals/icons.md [hunk 4]
```diff
@@ -2,22 +2,169 @@
...
+
+<div class="note note-note">
+  <h4>Tip</h4>
+  <p>A good method to solve an icon's metaphor during the design process is to show people without any context and see if they understand its meaning.</p>
+</div>
+
+## Drawing guidelines
+Our icon's illustration style is 2-dimensional without perspective using a single colour, without the illusion of a light source negating the need for highlights and shadows.
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: icon?

File: docs/visuals/icons.md [hunk 5]
```diff
@@ -2,22 +2,169 @@
...
+
+### Size
+A 24 × 24 point grid is used with a 2 point stroke weight for the paths. The grid to ensures they render sharp on low pixel density screens, while the stroke weight gives a good level of contrast to aid with our accessibility standards.
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: erroneous 'to' in 'The grid to ensures'

File: docs/visuals/icons.md [hunk 6]
```diff
@@ -2,22 +2,169 @@
...
+
+<img class="outline" alt="img" src="{{ site.baseurl }}/assets/img/icons/grid.png"/>
+
+### Padding
+The grid uses 2 pt padding to allow icons with unusual shapes to fit within it while maintaining a consistent size with the other icons. Although the grid is set at 24 × 24 points a typical icon's size is around 20 × 20 points.
+
+<img class="outline" alt="img" src="{{ site.baseurl }}/assets/img/icons/spacing.png"/>
+
+### Stroke
+When strokes are used to define shapes the weight is set at 2 pt with a centered path alignment and rounded terminal.
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: define shapes, the weight

File: docs/visuals/icons.md [hunk 7]
```diff
@@ -2,22 +2,169 @@
...
+
+<img class="outline" alt="img" src="{{ site.baseurl }}/assets/img/icons/stroke.png"/>
+
+
+### Pixel grid
+The coordinates of the path terminals should be a whole number to keep the path on the pixel grid.
+
+<img class="outline" alt="img" src="{{ site.baseurl }}/assets/img/icons/points.png"/>
+
+### Detail
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: Is this an opportunity to use the 'Do this' & 'Dont do this' green and red cards?

File: docs/visuals/icons.md [hunk 8]
```diff
@@ -2,22 +2,169 @@
...
+	table img {
+		width: 24px;
+		height: 24px;
+	}
 
-### List
-* [List item with link](http://bbc.com)
-* List item blank
-* [List item with link](http://bbc.com)
+	.markdown-body h3 {
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: can look to Origin padding utilities here perhaps

File: docs/visuals/icons.md [hunk 9]
```diff
@@ -2,22 +2,169 @@
...
+
+<img alt="img" src="{{ site.baseurl }}/assets/img/icons/export.png"/>
+
+### Naming convention
+As a guideline for naming icons, use the necessary properties from the following order and format:<br> <code>[object]-[status/function]-[variable].svg</code>
+<br>
+<br>
+For example <code>arrow-navigate-up.svg</code>, <code>lock-closed.svg</code> and <code>link</code> are acceptable. This allows us to easily group icons and catalog their functions.
+
+## Icon list
```
Comment (craigcwaveformlabs) [inline docs/visuals/icons.md]: _might_ want to look into using a Table for this content. although not sure if the .Table code has been brought across yet.

**Comment by craigcwaveformlabs** [comment]
@marcusfreeagent added a few comments mostly on proposed spelling changes. content looks good with as much as i know about icon development. main concern is over the inline CSS but we can fix that.

**Comment by marcusfreeagent** [comment]
Cheers @craigcwaveformlabs, I'd been meaning to chat to you about the CSS. I'll sort out the other stuff and we can sync up after.

**Comment by marcusfreeagent** [comment]
:+1:

**Pull request merged**